js/src/pubsub/unsubscribe.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env mocha */
 'use strict'
 
-const each = require('async/each')
+const eachSeries = require('async/each')
```

Comment:
This is actually using `async/each` here and not `eachSeries`. There is a typo on one side :)

Reply:
ha thanks